.changeset/light-chairs-happen.md

```diff
@@ -0,0 +1,27 @@
+---
+'@astrojs/react': minor
+'astro': minor
+---
+
+Make `.safe()` the default return value for actions. This means `{ data, error }` will be returned when calling an action directly. If you prefer to get the data while allowing errors to throw, chain the `.orThrow()` modifier.
```

Comment:
We should use this template for breaking changes inside experimental features 

https://contribute.docs.astro.build/docs-for-code-changes/experimental-feature-docs/#breaking-changes-while-still-experimental

Reply:
Ah ha, this is super helpful! I thought experimental packages followed the pre-1.0 rule:
- Major -> minor
- Minor -> patch

Didn't realize it was _always_ a patch. That makes life a bit simpler.

Reply:
Yes, experimental features are semver-free, so we quickly ship changes and fixes without problems.